schemas/app-sre/saas-file-target-1.yml

```diff
@@ -73,6 +73,18 @@ properties:
     required:
     - instance
     - name
+  image:
```

Comment:
Can you add some nice descriptions here documenting the new fields for future reference?